src/screens/settings/SettingsCustomCodeScreen/Components/SimpleCodeEditor.tsx

```
@@ -0,0 +1,591 @@
+  style,
+  hide,
+  setLines,
+  isDark = false,
+  startLine = 0,
+}: MemoizedHighlightedCode) {
+  const opacityStyle = useMemo(() => extractOpacityStyle(style), [style]);
+  const textStyle = useMemo(() => extractTextStyle(style), [style]);
+  const contentPadding = useMemo(() => extractContentPadding(style), [style]);
+  const lineHeight = useMemo(() => getLineHeight(style), [style]);
+  if (!lines) {
+    lines = useStableLineModels(value!);
```

ESLint

[error] React Hook "useStableLineModels" is called conditionally. React Hooks must be called in the exact same order in every component render.

(react-hooks/rules-of-hooks)